src/Settings.php

```diff
@@ -444,6 +445,33 @@ protected function createLanguageSection($settings_key)
         }
         /** END LANGUAGE SECTION */
 
+        /** STYLING SECTION */
+        protected function createStylingSection($settings_key)
+        {
+            global $ADMWPP_LANG;
```

Comment:
Do we need this?

Reply:
No we need it in this function, Will remove it.